src/services/cleanFormSubmissionModel.ts

```diff
@@ -187,12 +187,20 @@ function cleanElementValue(
           break
         }
         if (stripBinaryData) {
+          const checklistObject = submission[element.name] as ComplianceValue
+          const value = checklistObject?.notes?.trim()
```

Comment:
This stuff should be done before the `if` statement so we aren't repeating it in the `if` and `else`